tests/test_restaurant_item.py

```diff
@@ -0,0 +1,32 @@
+import os
+import json
+
+
+def test_get_nonexisten(test_client):
+    response = test_client.get('/api/restaurants/3000')
+    assert response.status_code == 404
+    assert b'{\n  "description": "Restaurant with such ID doesn\'t exist", \n  "name": "Not Found"\n}\n' \
+           == response.data
+
+
+def test_put_nonexisten(test_client):
+    response = test_client.put('/api/restaurants/3000', json={"name": "Hotfix"})
+    assert response.status_code == 404
+    assert b'{\n  "description": "No restaurant to update. Restaurant with such ID doesn\'t exist", ' \
```

Comment:
the order in which JSON is encoded is not fixed. instead, it is better to very body correctness, here is how Flask suggests to test JSON APIs: https://flask.palletsprojects.com/en/1.1.x/testing/#testing-json-apis

Reply:
fixed